simulation/src/test/resources/report.md

```diff
@@ -29,7 +29,7 @@ one_endpoint_dies_on_each_server[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	succe
         slow_503s_then_revert[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=100.0%	client_mean=PT0.075948S    	server_cpu=PT3M47.844S    	client_received=3000/3000	server_resps=3000	codes={200=3000}
             slow_503s_then_revert[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT0.707925608S 	server_cpu=PT34M27.664333313S	client_received=3000/3000	server_resps=3390	codes={200=3000}
                       slow_503s_then_revert[UNLIMITED_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT1.470730224S 	server_cpu=PT1H11M35.535333313S	client_received=3000/3000	server_resps=3825	codes={200=3000}
-slowdown_and_error_thresholds[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=40.4%	client_mean=PT1M36.106700399S	server_cpu=PT10H20M47.626666524S	client_received=10000/10000	server_resps=10000	codes={200=4043, 500=5957}
+slowdown_and_error_thresholds[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=1.4%	client_mean=PT25.998591133S	server_cpu=PT10H38M15.47999998S	client_received=10000/10000	server_resps=10000	codes={200=136, 500=9864}
```

Comment:
Ok I straight up do not understand why the last one had 40% success and now we have 1% :/